dts/arm/nordic/nrf51822.dtsi

```diff
@@ -133,6 +134,18 @@
 			label = "SPI_1";
 		};
 
+		/* nRF TEMP Peripheral:
+		 * .. Default enabled,
+		 * .. Interrupt priority:
+		 * .. .. Range: 0-1, Default: 1
+		 */
+		temp: temp@4000C000 {
```

Comment:
But this line stays UPPER case C? Or lower case as well?

Reply:
I believe both occurrences of the address should be corrected to lowercase. As it was done in the commit I mentioned.

Reply:
Ok, then we should also change the flash-controller@4001E000 of this file as well i suppose? 
I don't understand why we don't accept both uppercase and lowercase (assuming this is a tool "issue"), but I will make the change for this temp node.

Reply:
Changed TEMP node addresses to lower case.